axon-framework/deadlines/deadline-managers.md

```diff
@@ -52,6 +52,11 @@ Note that there are more options to cancel a deadline next to the previously men
 
   This allows canceling a deadline by name from differing scopes then the one it's executed in.
 
+> **Caveats for the JobRunr implementation.**
+> 
+> Since JobRunr has no way to search for search deadlines, besides by id, all of the `cancelAll` methods are not implemented for the `JobRunrDeadlineManager`.
```

Comment:
```suggestion
> Since JobRunr has no way to search for deadlines, besides by id, all of the `cancelAll` methods are not implemented for the `JobRunrDeadlineManager`.
```